libraries/AP_Stats/AP_Stats.h

```diff
@@ -17,7 +17,7 @@ class AP_Stats
     // init() to set initial values from the parameters!
     uint32_t flttime; // seconds in flight (or driving)
     uint32_t runtime; // total wallclock time spent running ArduPilot (seconds)
-    uint32_t reset; // last time parameters were reset
+    uint32_t reset;   // last time AP_Stats parameters were reset (in seconds since AP_Stats Jan 1st 2016)
```

Comment:
Why the second AP_Stats on this line?

Reply:
The second thime it is meant as the epoch name